src/com/lftechnology/java/training/dipak/Test.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.dipak;

```

Comment:
While sending pull request, include the question. Pull request name should be meaningful.